enhancements/sig-architecture/64-placementStrategy/README.md

```diff
@@ -0,0 +1,220 @@
+  labels:
+    cluster.open-cluster-management.io/placement: operator-placement
+    cluster.open-cluster-management.io/clustergroupindex: 2
+status:
+  decisions:
+    - clusterName: cls155
+      reason: ''
+    ...
+```
+
+The Placement Helper [library](https://github.com/open-cluster-management-io/api/blob/main/cluster/v1beta1/helpers.go) should provide functions to facilitate retrieve the cluster groups.
+
+
+### How Workload applier APIs will benefits from using placementStrategy
+
+The OCM APIs such as ManifestWorkReplicaSet should check the PlacementStatus APIs to determine if the workloads need to be rolled out progressively. For example; the ManifestWorkReplicaSet API has a reference to a [placement](https://github.com/open-cluster-management-io/api/blob/main/work/v1alpha1/types_manifestworkreplicaset.go#L64) that will be used to apply the manifestWorkTemplate. The ManifestWorkReplicaSet controller should be able to determine based on the `placement->status->numberOfGroups` how to progressively roll out the manifestWorks. The ManifestWorkReplicaSet controller will apply the manifestWork to the canary clusters group first by selecting all placementDecisions with label ClusterGroupIndex equal to 0 and wait till the manifestWorks become in available state at the canary clusters. Then the ManifestWorkReplicaSet controller will apply the manifestWorks to the next cluster group by select placementDecisions with label ClusterGroupIndex equal to 1 and wait till the manifestWorks become in available state. Then do the same for the rest of cluster groups.
+
+
+### Risks and Mitigation
+
+* Security is based on the placement resources which bind to ManagedClusterSets and the namespace where the placement CR is created.
+
+### Test Plan
+
+- Unit tests will cover the new changes for the APIs.
+- Integration tests will cover all user stories.
+- And e2e tests will cover the following cases.
+  - Create/updated/delete placement with placementStrategy.
+  - Placement Decision changes while more managed clusters selected by the placement.
+  - Placement Decision changes when update the placementStrategy.
+
+### Graduation Criteria
+**Note:** *Section not required until targeted at a release.*
+
+#### Tech Preview -> GA
+**For non-optional features moving to GA, the graduation criteria must include
+end to end tests.**
+
+### Upgrade / Downgrade Strategy
+
+Placement->placementStrategy API will be supported on the target release of OCM (open cluster management) and it should not affect the OCM current APIs functionalities or back compatibility.
+
+## Alternatives
+
+1. End user can create many placements and apply the workloads using each placement one by one to have a progressive roll out. 
+1. Each OCM APIs such as ManifestWorkReplicaSet, ClusterManagementAddon and Policy define its own roll out strategy to determine the progressive roll out.
+1. Current alternative does not satisfy the requirements as explained above in the motivation section
```

Comment:
with multiple placements, we can not define the number of clusters or percentage in each group, and dynamically generate each groups

Reply:
Agreed.